### DISCUSSION TITLE
Is there any way to list all contents of a folder and sub-folders in OneDrive?

### DISCUSSION THREAD

**Discussion by Subhajit97** [microsoftgraph/msgraph-sdk-go#238] [Q&A] [answered]
Hi!
I am trying to get all the contents of a folder and sub-folders in my OneDrive. I checked the API documentation where we can pass the `item-id` or the `path-relative-to-root` (if known) to get the contents.
https://docs.microsoft.com/en-us/graph/api/driveitem-list-children?view=graph-rest-1.0&tabs=http#http-request
But is there any API available using which I can list all the contents without passing ids?

**Comment by baywet** [comment, marked as answer]
Hi @Subhajit97 
Issues and discussions on this repository are reserved to topics related to the SDK itself. For any API specific question, the best thing is to post to [the appropriate Microsoft Q&A forum](https://docs.microsoft.com/en-us/answers/topics/microsoft-graph-files.html) where members of the service team will be able to answer.
